docs/src/api/reference.md

```
@@ -18,6 +18,7 @@ Total: 2834 entries across 119 modules.
 - [`better-sqlite3`](#better-sqlite3)
 - [`bignumber.js`](#bignumber-js)
 - [`buffer`](#buffer)
+- [`bun:ffi`](#bun-ffi)
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)